.eslintrc

```diff
@@ -19,6 +19,8 @@
     "jsx-a11y/no-static-element-interactions": ["off"],
     "react/forbid-prop-types": ["error", { "forbid": ["any"] }],
     "react/jsx-filename-extension": ["error", { "extensions": [".js"] }],
-    "react/require-default-props": ["off"]
+    "react/require-default-props": ["off"],
+    "global-require": ["off"],
```

Comment:
instead of switching off rules completely because you're breaking them in one file, just disable it in that one file only.